src/serve_rendered.js

```diff
@@ -263,6 +263,18 @@ module.exports = function(options, repo, params, id, publicUrl, dataResolver) {
     styleJSON.glyphs = 'fonts://' + styleJSON.glyphs;
   }
 
+  (styleJSON.layers || []).forEach(function(layer) {
```

Comment:
Looks like they have a much simpler way to remove 3d buildings. We try to retain them.

Reply:
Looks like our effect will still go ahead, as the fill-extrusion layers are still there.